config/consts.js

```diff
@@ -9,4 +9,9 @@ module.exports = {
         This is used as a default amount to configure max lending amount.
      */
     DEFAULT_MAX_AMOUNT: 1000,
+
+    /**
+     * Represents a fake Node Component version.
+     */
+    DUMMY_NODE_COMPONENT_VERSION: 123,
```

Comment:
I would set the initial version directly (not a dummy value) describing the format. Maybe 1.02.03.04 => 1020304?

Reply:
I set 1000000 representing 1.00.00.00 but I am confirming this with Esteban which is going to be the initial version of each component.